common/web/input-processor/src/build.sh

```diff
@@ -63,7 +63,7 @@ if [ $BUILD_LMLAYER = true ]; then
     echo ""
     echo "Compiling the Language Modeling layer module..."
     ./build.sh $FLAGS || fail "Failed to compile the language modeling layer module."
-    cd ../core/web/input-processor/src
+    cd ../web/input-processor/src
```

Comment:
... not saying it's your fault; I know the `git blame` likely rests on me.

Just... this really should be a `pushd` / `popd` setup instead.

Reply:
Sure. But that doesn't need to happen in this PR.